src/asmcnc/comms/serial_connection.py

```diff
@@ -639,6 +639,9 @@ def end_stream(self):
 
         log("Ending stream...")
 
+        # Always switch to mm, in case the job put you in inches
+        self.write_command('G21')
```

Comment:
Would be good to have this (for both unit options) as a readable router_machine function, like zUp, spindle_off etc.